packages/suite/src/views/recovery/index.tsx

```diff
@@ -92,12 +92,12 @@ export const Recovery = ({ onCancel }: ForegroundAppProps) => {
         actions.checkSeed();
     };
 
-    const model = device && getDeviceModel(device) === '1' ? '1' : 'T';
+    const deviceModel = getDeviceModel(device);
     const learnMoreUrl = getCheckBackupUrl(device);
-    const isModelOne = model === '1';
-    const statesInProgressBar = isModelOne
-        ? ['initial', 'select-word-count', 'select-recovery-type', 'in-progress', 'finished']
-        : ['initial', 'in-progress', 'finished'];
+    const statesInProgressBar =
+        deviceModel === DeviceModel.T1
```

Comment:
nit: I might be wrong but is that related to `Capability_PassphraseEntry`?

Reply:
nah it is related to `Capability_RecoveryEntry` which is not yet implemented